src/commcare_cloud/ansible/roles/postgresql/templates/pgbouncer_hba.conf.j2

```diff
@@ -14,7 +14,7 @@ host{% if postgresql_ssl_enabled %}ssl{% endif %}    {{ db.name }}    {{ user.us
  {% endif %}
 {% endfor %}
 {%  else %}
-{% for host in (groups['formplayer'] + groups.get('touchforms', []) + groups['webworkers'] + groups['celery'] + groups['pillowtop'] + groups['postgresql'] + groups('pg_standby',[]) + groups('airflow', [])) | unique %}
+{% for host in (groups['formplayer'] + groups.get('touchforms', []) + groups['webworkers'] + groups['celery'] + groups['pillowtop'] + groups['postgresql'] + groups.get('pg_standby',[]) + groups.get('airflow', [])) | unique %}
 # {{ host }}
 {% for db_name, db_list in postgresql_dbs.all|groupby('name') %}
 {% set db = db_list[0] %}
```

Comment:
This will still be an issue for DB's that exist on multiple hosts since it's only taking the first one in the list. Can fix separately though

Reply:
Hey @NitigyaS, can you grab this if/when you have a chance. This was an issue last night, since we had two hosts pgucr and pgnondashboarducr0 having same DB `commcare_icds_aggregatedata`. I had to manually update the config as this wasn't working.